plugins/scaffolder-backend/api-report.md

```diff
@@ -391,6 +392,8 @@ export const createPublishGithubPullRequestAction: ({
   targetPath?: string | undefined;
   sourcePath?: string | undefined;
   token?: string | undefined;
+  reviewers?: string[] | undefined;
```

Comment:
I wonder if it makes sense just to have one option here, and then call the github API's for each one and check to see if it's a team or a user and then add accordingly?

Reply:
I think it isn't a good idea for a couple of reasons:
- a team can potentially have the same name of a GitHub user
- it slows down the process by calling the API for a potentially non-trivial number of times (what if you have 5+ reviewers?)

Reply:
> No, they cannot. Pretty sure they always need to be prefixed by the organization name anyways when adding to remove this ambiguity.

I can assure you that you can pass team names without any prefix to the `requestReviewers` API and it works, since we do it in one of our Backstage actions 😊